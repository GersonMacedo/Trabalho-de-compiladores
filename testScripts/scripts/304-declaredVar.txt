! -s CONTENT --disable-logs printer
! should accept the usage of declared vars
program p ;
    var a : integer;
begin
    a := 1;
end .