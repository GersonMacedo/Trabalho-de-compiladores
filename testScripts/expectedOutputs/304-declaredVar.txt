
Performing up to content analysis at testScripts/scripts/304-declaredVar.txt
No errors found
